Projects/UOContent/Items/Weapons/Abilities/ForceOfNature.cs

```diff
@@ -0,0 +1,118 @@
+            defender.FixedParticles(0x374A, 1, 17, 9502, 1108, 4, (EffectLayer)255);
+
+            Remove(attacker);
+
+            ForceOfNatureTimer t = new ForceOfNatureTimer(attacker, defender);
+            t.Start();
+
+            _table[attacker] = t;
+        }
+
+        private static readonly Dictionary<Mobile, ForceOfNatureTimer> _table = new Dictionary<Mobile, ForceOfNatureTimer>();
+
+        public static void Remove(Mobile m)
+        {
+            if (_table.Remove(m, out var timer))
+            {
+                timer.Stop();
+            }
+        }
+
+        public static void OnHit(Mobile from, Mobile target)
+        {
+            if(_table.TryGetValue(from,out var t))
+            {
+                t.Hits++;
+                t.LastHit = Core.Now;
+
+                if (t.Hits % 12 == 0)
+                {
+                    int duration = target.Skills[SkillName.MagicResist].Value >= 90.0 ? 1 : 2;
+                    target.Paralyze(TimeSpan.FromSeconds(duration));
+
+                    target.FixedEffect(0x376A, 9, 32);
+                    target.PlaySound(0x204);
+
+                    t.Hits = 0;
+
+                    from.SendLocalizedMessage(1004013); // You successfully stun your opponent!
+                    target.SendLocalizedMessage(1004014); // You have been stunned!
+                }
+            }
+        }
+
+        public static double GetDamageScalar(Mobile from, Mobile target)
+        {
+            if(_table.TryGetValue(from, out var t))
+            {
+                if (t.Target == target)
+                {
+                    double bonus = Math.Min(100, Math.Max(50, from.Str - 50));
+
+                    return (100 + bonus) / 100;
+                }
+            }
+
+            return 1.0;
+        }
+
+        private class ForceOfNatureTimer : Timer
+        {
+            private readonly Mobile m_Target, m_From;
+
+            private DateTime m_LastHit;
+            private int m_Hits;
+
+            public Mobile Target => m_Target;
+            public Mobile From => m_From;
+            public int Hits { get { return m_Hits; } set { m_Hits = value; } }
+            public DateTime LastHit { get { return m_LastHit; } set { m_LastHit = value; } }
+
+            public ForceOfNatureTimer(Mobile from, Mobile target)
+                : base(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(1))
+            {
+                m_Target = target;
+                m_From = from;
+                m_Hits = 1;
+                m_LastHit = Core.Now;
+            }
+
+            protected override void OnTick()
+            {
+                if (!m_From.Alive || !m_Target.Alive || m_Target.Map != m_From.Map || m_Target.GetDistanceToSqrt(m_From.Location) > 10 || m_LastHit + TimeSpan.FromSeconds(20) < Core.Now || Index > 36)
+                {
+                    Remove(m_From);
+                    return;
+                }
+
+                if (Index == 1)
```

Comment:
Is this correct? If I understand the logic correctly, there is damage on Index == 1 (the second tick), but never again?

Reply:
This is correct, i added baseweapon code in next push. First index does the damage. Then mobile and timer still lies in Table which is connected to BaseWeapon Onhit. On Index 36 its deleted if no condition before removes it.